code/game/objects/objs.dm

```diff
@@ -46,6 +46,9 @@
 	/// broadcasted to as long as the other guys network is on the same branch or above.
 	var/network_id = null
 
+	///List of object types that the item can be anchored on top of, such as a table
+	var/list/anchorables = null
```

Comment:
Make this a `string_list()` during initialization to reduce duplicate lists.

Reply:
using the string_list now, thanks.